### PR TITLE
example(dataset): add cityscapes 3dbox dataset

### PR DESCRIPTION
**Pull request by anda-ren**
- add cityscapes 3dbox dataset
- add 3d box in sw
- fix datastore object merge None bug
image

## Modules
- [ ] UI
- [ ] Controller
- [ ] Agent
- [ ] Client
- [ ] Python-SDK
- [ ] Others

## Checklist
- [ ] run code format and lint check
- [ ] add unit test
- [ ] add necessary doc

### CODE REVIEW

**Comment by tianweidut** [comment]
@anda-ren  add some docs for `BoundingBox3D`?

File: client/starwhale/api/_impl/data_store.py [hunk 1]
```diff
@@ -417,6 +417,8 @@ def __init__(self, raw_type: Type, attrs: Dict[str, SwType]) -> None:
         self.attrs = attrs
 
     def merge(self, type: SwType) -> SwType:
+        if type is UNKNOWN or self is type:
```
Comment (tianweidut) [inline client/starwhale/api/_impl/data_store.py:420]: add some specified unit tests for this change in the test_datastore.py?

**Comment by anda-ren** [comment]
> @anda-ren add some docs for `BoundingBox3D`?

added

File: client/starwhale/api/_impl/data_store.py (hunk 1 above)
Comment (anda-ren) [reply client/starwhale/api/_impl/data_store.py:420]: > add some specified unit tests for this change in the test_datastore.py?

added

**Review by tianweidut — APPROVED** [review]
LGTM

**Pull request merged**